src/q5/stat-FSAB1105/exam/2018/Septembre/All/stat-FSAB1105-exam-2018-Septembre-All.tex

```diff
@@ -0,0 +1,201 @@
+
+\begin{enumerate}
+    \item Réalisez un test pour déterminer si la variance diffère en fonction du sexe.
+    \item Donnez un intervalle de confiance pour $\mu_H - \mu_F$. Enoncez précisément les hypothèses sous-jacentes à votre calcul et effectuez le lien avec le point 1.
+    \item La vitesse moyenne diffère-t-elle en fonction du sexe?
+\end{enumerate}
+
+\begin{solution}
+
+\begin{enumerate}
+    \item On réalise le test d'hypothèse bilatéral $H_0 : \sigma^2_F = \sigma^2_H$ vs $H_1 : \sigma^2_F \neq \sigma^2_H$.
+
+    Le test statistique de \textit{Fisher}\footnote{Il faut mettre la valeur la plus grande au numérateur.} donne \( f = s_F/s_H = 1.1547 \). On rejette $H_0$ si 
+    \[ f < F_{n_1-1,n_2-1,1-\alpha/2} = F_{15,20,0.975} = 0.4292 \qquad\mathrm{ou}\qquad f > F_{n_1-1,n_2-1,\alpha/2} = F_{15,20,0.05} = 2.2 \]
+
+    La valeur observée est en dehors de la région de rejet, donc on ne peut pas rejeter l'hypothèse nulle.
+
+    \item On suppose avoir deux échantillons issus d'une distribution normale, d'espérances \( \mu_F \) et \( \mu_H \) avec la même variance\footnote{C'est légitime car on n'a pas rejeté cette hypothèse au point 1.}. Puisque cette variance est inconnue, on utilise une distribution \textit{student} pour donner un intervalle de confiance bilatéral à \SI{95}{\percent}:
+    \[ CI = (\bar{x}_H - \bar{x}_F) \pm t_{n_H+n_F-2,\alpha/2}\:\sqrt{\frac{(n_H-1){s_H^2}+(n_F-1)s_F^2}{n_H+n_F-2}}\sqrt{\frac{1}{n_H}+\frac{1}{n_F}
+    } = \left[0.307;5.693\right] \]
+    \item Non, car 0 n'est pas dans l'intervalle de confiance pour $\mu_H - \mu_F$ (point 2). On ne peut donc pas rejeter l'hypothèse nulle pour ce cas ($H_0:\mu_H = \mu_F$) en déclarant que les vitesses sont différentes.
+\end{enumerate}
+
+\end{solution}
+
+\section{(4 points)}
+On peut modéliser la durée d'une connexion sur le site \textit{www.coolsite.be} par une loi de densité
+$$\theta^{-2}xe^{-x/\theta}I(x>0).$$
+Pour fixer vos tarifs publicitaires, vous voulez estimer le paramètre $\theta$ à partir d'un échantillon iid $X_1,\dots,X_n$ de $n$ durées de connexions. On vous donne $\mathbb{E}[X_i] = 2\theta$ et $\mathbb{V}[X_i] = 2\theta^2$.
+\begin{enumerate}
+    \item Calculez l'estimateur de maximum de vraisemblance (MLE) $\hat{\theta}_n$ de $\theta$.
+    \item Calculez l'erreur quadratique moyenne (MSE) de $\hat{\theta}_n$?.    \item Proposez un estimateur $\hat{\lambda}_n$ consistant pour $\lambda = \ln(\theta)$ en justifiant votre choix.
+    \item Donnez la distribution asymptotique de $\hat{\lambda}_n$.
+\end{enumerate}
+
+\begin{solution}
+
+\begin{enumerate}
+    \item Use the MLE's function, than it's log function:
+    \[ LL_n(\theta) = \sum_i \ln\left( f(x_i;\theta)\right) = \sum_i -2\ln(\theta)+\ln(x_i)-\frac{x_i}{\theta} = -2n\ln(\theta)+\sum_i\ln(x_i)-\frac{\sum_ix_i}{\theta} \]
+    Maximize the parameter, that is derive the log function along the parameter. The derivative is 0 since you want to maximize the parameter.
```

Comment:
that is,*

"along the parameter" -> "with respect to the parameter" (je crois que c'est ce que tu veux dire).